bin/proving-service/README.md

```diff
@@ -28,48 +28,49 @@ This will spawn a worker using the hosts and ports defined in the command option
 
 ## Proxy
 
-First, you need to create a configuration file for the proxy with:
+To start the proxy service, you will need to run:
 
 ```bash
-miden-proving-service init
+miden-proving-service start-proxy [worker1],[worker2],...,[workerN]
+miden-proving-service start-proxy 0.0.0.0:8084 0.0.0.0:8085
 ```
```

Comment:
nit: I would separate these like so:

------
To start the proxy service, you will need to run:
```bash
miden-proving-service start-proxy [worker1],[worker2],...,[workerN]
```
For example:
```bash
miden-proving-service start-proxy 0.0.0.0:8084 0.0.0.0:8085
```